src/pages/profile/index.tsx

```diff
@@ -81,6 +118,20 @@ export default function Profile() {
                   GitHub
                 </Link>
               </div>
+              {checked && (
+                <GitHubCalendar
+                  username={data.user.login}
+                  transformData={selectLastHalfYear}
+                  blockSize={10}
+                  fontSize={10}
+                  loading={!data}
+                  colorScheme="light"
+                  hideColorLegend
+                  labels={{
+                    totalCount: "{{count}} contributions in the last half year",
```

Comment:
```suggestion
                    totalCount: "{{count}} contributions in the last 6 months",
```

I think with "6 months" is more readable

Reply:
lgtm